frontend/src/components/hub/LoggedOutLocationHubBox.tsx

```
@@ -168,17 +174,25 @@ export default function LoggedOutLocationHubBox({ headline, isLocationHub, locat
-          <div className={classes.advantagesBox}>
-            {REASONS_TO_JOIN.map((r) => (
-              <ReasonToJoin reason={r} />
-            ))}
-          </div>
+          <Typography component="p">
+            {(texts as any)[(subHub ? (subHub as string) : (parentHub as string)) + "_welcometext"]}
+          </Typography>
+          {!subHub && (
+            <div className={classes.advantagesBox}>
+              {REASONS_TO_JOIN.map((r) => (
+                <ReasonToJoin reason={r} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)